### PR TITLE
Capstone 1

### PR DESCRIPTION
**Pull request by HSMathebula**
### Capstone 1 -Web development course website

In this Project I:

- Add the home page and about with HTML.
- Add style for the home page and about.
- Style two media queries 375px for mobile 768px for desktop.
- Record a video presenting my features. video link - https://www.loom.com/share/93c2eecbb7bb424593f625ef7927ddea

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                    <ul>
+                        <li class="p-cta active"><a href="about.html">About</a></li>
+                        <li class="p-cta"><a href="#">Program</a></li>
+                        <li class="p-cta"><a href="#">join</a></li>
+                        <li class="p-cta"><a href="#">Sponsor</a></li>
+                        <li class="p-cta"><a href="#">News</a></li>
+                        <li class="p-cta call2action"><a href="#">Start Now</a></li>
+                    </ul>
+                </div>
+            </nav>
```
Comment (Whoistolu) [inline about.html:49]: - [ ] [REQUIRED] For the `navbar`, kindly justify the contents, ensured that the contents are properly aligned just as it is on the [behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

### Your Implementation
Screenshot from 2022-04-21 17-12-25

### Behance Design
Screenshot from 2022-04-21 17-12-14

File: index.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                </div>
+                <p class="program-disc">Watch some of our meterials from various expect across the world.</p>
+            </div>
+            <div class="card g-col-12 g-col-md-2">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/board.png" alt="lecture image">
+                    <p class="t1">CC Exhibition</p>
+                </div>
+                <p class="program-disc">Our board contain timetables and announcements.</p>
+            </div>
```
Comment (Whoistolu) [inline index.html]: -  [ ] [REQUIRED] Kindly note that the cards need not be separated by little spaces each, just as it is on the behance design.
### Your Implimentation
Screenshot from 2022-04-21 17-21-56

### Behance Design
Screenshot from 2022-04-21 17-21-55

File: index.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        </div>
+        <p class="whole-program font-face">SEE THE WHOLE PROGRAM</p>
+    </section>
+    
+    <!-- speaker section -->
+    <section class="container main-speakers">
+        <div class="p-cont">
+            <p class="program-title">Featured Speakers</p>
+            <hr>
+        </div>
```
Comment (Whoistolu) [inline index.html:117]: - [ ] [REQUIRED] On the mobile homepage, kindly remove the excess space you have between the `Start Learning` button and `Featured Speaker` header so your design can look more professional.
 #### Your Implementation
Screenshot from 2022-04-21 17-28-33

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @HSMathebula 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    </section>
+
+    <!-- footer -->
+    <footer>
+        <p class="f-logo">CodeWorld</p>
+        <div class="footer-text">
+            <span>2022 CodeWorld, some rights reserved.</span>
+        </div>
+    </footer>
```
Comment (Wes-Isaac) [inline index.html:148]: - [ ] [_OPTIONAL_] your implementation of the footer is a bit different than that of the design, kindly fix that.

### Behance Design
image

### your implementation
image

Please try to reduce the space.

File: README.md [hunk 5]
```diff
@@ -1,56 +1,50 @@
...
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Live Demo Link](https://hsmathebula.github.io/)
```
Comment (Wes-Isaac) [inline README.md:17]: - [ ] [_OPTIONAL_] Please upload your site with github pages and update this link.

**Review by Wes-Isaac — APPROVED** [review]
Hi @HSMathebula, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**